### PR TITLE
doc: prepare new release 4.1.2

### PR DESCRIPTION
**Pull request by raphj**
#5412

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done - nothing to test

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 4.1.2 / 2022-02-14
+
+This maintenance release fixes a few things:
+ - the Docker image, broken after a setuptools update, once again works
```
Comment (inkhey) [inline changelog.md:4]:
```suggestion
 - the experimental Docker image for arm64, broken after a setuptools update, once again works
```

**Review by inkhey — APPROVED** [review]

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+ - on mobile devices, a work around prevents the send button to be covered by the browser's native UI
```
Comment (Relf-Algoo) [inline changelog.md:5]:
```suggestion
 - on mobile devices, a workaround prevents the send button to be covered by the browser's native UI
```

**Review by Relf-Algoo — CHANGES_REQUESTED** [review]
very small change request. Otherwise ok.

File: backend/setup.cfg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = tracim_backend
-version = 4.1.1
+version = 4.1.2
```
Comment (knodelle) [inline backend/setup.cfg:3]: don't we need to mention the build as well?

**Review by knodelle — APPROVED** [review]

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
+# 4.1.2 / 2022-02-14
```
Comment (knodelle) [inline changelog.md:1]: make sure you update the date here @raphj

Comment (raphj) [reply changelog.md:1]:
```suggestion
# 4.1.2 / 2022-02-15
```

**Pull request merged**